src/components/TodoList/TodoList.tsx

```diff
@@ -16,85 +28,49 @@ export const TodoList: React.FC = () => (
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+      {todos.map(({ id, title, completed }) => (
+        <tr
+          data-cy="todo"
+          key={id}
+          className={cn({
+            'has-background-info-light': selectedTodoId === id,
+          })}
+        >
+          <td className="is-vcentered">{id}</td>
+          <td className="is-vcentered">
+            {completed && (
+              <span className="icon" data-cy="iconCompleted">
+                <i className="fas fa-check" />
+              </span>
+            )}
+          </td>
+          <td className="is-vcentered is-expanded">
+            <p className={cn({
+              'has-text-success': completed,
+              'has-text-danger': !completed,
+            })}
+            >
+              {title}
+            </p>
+          </td>
+          <td className="has-text-right is-vcentered">
+            <button
+              data-cy="selectButton"
+              className="button"
+              type="button"
+              onClick={() => selectTodo(id)}
+            >
+              <span className="icon">
+                <i className={cn('far', {
+                  'fa-eye': id !== selectedTodoId,
+                  'fa-eye-slash': id === selectedTodoId,
```

Comment:
same here

Reply:
Hi Nazar I added it this way: 
```
className={cn('far fa-eye', {
                  'fa-eye-slash': id === selectedTodoId,
                })}
```
but it fails me in tests. Because tests are checking that I far-eye class shouldn' t exist when far-eye-slash class is there.